lib/linux.js

```diff
@@ -118,29 +119,15 @@ function _createScripts ( linux ) {
 }
 
 /**
- * Copy the app in the given folder
```

Comment:
In fact, we should not copy — fpm allows you to map any dir to any path. See https://github.com/jordansissel/fpm/wiki/Source:-dir